contracts/contracts/interfaces/IMetadataPointer.sol

```diff
@@ -0,0 +1,7 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity ^0.7.6;
+
+struct MetaPtr {
+  uint256 protocol; // mapping from integer to protocol
```

Comment:
nit: word "mapping" in this comment is a little misleading/confusing. Might say something like "integer code which corresponds to a specific storage protocol".

Reply:
Heh, fair enough. I'll standup the repo mentioned in the description first and link to it here + rephrase for clarity

Reply:
Done in 1a0b682